ecllibrary/std/Uni.ecl

```diff
@@ -382,4 +382,19 @@ EXPORT unsigned4 WordCount(unicode text, varstring localename = '') :=
 EXPORT unicode GetNthWord(unicode text, unsigned4 n, varstring localename = '') :=
     lib_unicodelib.UnicodeLib.UnicodeLocaleGetNthWord(text, n, localename);
 
+/**
+ * Returns everything except the nth word and, depending on the nth word, some whitespaces from the string.  Words are marked by the unicode break semantics.
```

Comment:
minor: We generally aim to have lines less than 120 characters.  Not always enforced, but it would be worth reformatting this so the long lines were a bit shorter.